### DISCUSSION TITLE
Announcement of the documentation site, and calling for requests.

### DISCUSSION THREAD

**Discussion by vrtmrz** [vrtmrz/obsidian-livesync#185] [Announcements]
I am currently working on the documentation. This would not just be an item description like in the readme but would include how it works, what settings are recommended, etc. However, I think that it has been a bit too greedy, and I am completely lost. (My dears, you have beheld **some** ruins.)

For long days I had been lost, but I decided to back to the origin. And thanks to the opportunity, I now have the site that is hosted at `Obsidian Publish` for documentation. The site is [here](https://docs.vrtmrz.net/Welcome). (Yes, I am swallowing my pride.)

Having said that, I still don't know what my priorities are for documentation, and I'm afraid I'm getting lost again. To avoid that, I thought that I should ask you all for the order.

I have created a new Discussion; where to write or vote on what you would like to be explained. After having published the entry, I will post the URI as an answer to that. Any format is acceptable, but one question for one discussion is preferable. Short questions are also welcome, I think short documents will be published quickly.

I am looking forward to receiving your requests.

[Documentation requests](https://github.com/vrtmrz/obsidian-livesync/discussions/categories/documentation-requests)

**Comment by hilsonp** [comment]
This is a great initiative.

I think the most important would be:

1. How to setup synchronisation between several devices (sunny case).
2. How to elect one master device and delete/overwrite CouchDB and other devices with it.
3. Description of each setting/button/paletteAction.
